projects/js-packages/config/README.md

```diff
@@ -0,0 +1,29 @@
+# Jetpack Config
```

Comment:
I think we should add another section on the package API (`jetpackConfigHas`, `jetpackConfigGet`).
What do you think?

Reply:
added